addons/sale/sale.py

```diff
@@ -175,6 +143,36 @@ def _resolve_team_id_from_context(self, cr, uid, context=None):
                 return int(team_ids[0][0])
         return None
 
+    def _count_all(self, cr, uid, ids, field_name, arg, context=None):
+        res = {}
+        for order in self.browse(cr, uid, ids, context=context):
+            tot = 0.0
+            invoice_existence = False
+            res[order.id] = {
+                'invoice_count': len(order.invoice_ids),
+                'invoiced': True,
+                'invoiced_rate': 0.0
+            }
+            for invoice in order.invoice_ids:
+                if invoice.state != 'cancel':
+                    invoice_existence = True
+                    if invoice.state != 'paid':
+                        res[order.id]['invoiced'] = False
+                        break
+            if not invoice_existence or order.state == 'manual':
+                res[order.id]['invoiced'] = False
+            if res[order.id]['invoiced']:
```

Comment:
here you have to check with order.invoiced . because res[order.id]['invoiced'] you already set default true..which will display wrong result here.